delete_docker_images.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+X=`docker images | grep none | awk '{ print $3}'`
+for i in $X; do   docker rmi $X; done
```

Comment:
You could do this in one with:
docker rmi -f $(docker images -f "dangling=true" -q)